readme/rich_text_editor.md

```diff
@@ -22,4 +22,6 @@ However **there is a catch**: in Joplin, notes, even when edited with this Rich
 
 - If a note is of 'Markup - Markdown' and contains HTML formatting, this may be lost when editing in the Rich Text editor as it cannot be converted to Markdown. Notes of 'Markup - HTML' are not affected by edits in the Rich Text editor as this conversion does not take place.
 
+- All reference links ([title][link-name]) are converted to inline links ([title](https://example.com)) when Joplin saves changes from the Rich Text editor.
```

Comment:
Let's use a inlide code for the examples...

```suggestion
- All reference links (`[title][link-name]`) are converted to inline links (`[title](https://example.com)`) when Joplin saves changes from the Rich Text editor.
```

Reply:
and maybe add somthing like:

----
This also means that all your reference links will be gone, if you switch from the markdown editor to the rich text editor.
----

Unless the rich text editor doesn't automatically save, but I think it does.